openvdb_houdini/SOP_OpenVDB_Scatter.cc

```diff
@@ -186,6 +186,19 @@ newSopOperator(OP_OperatorTable* table)
             "For scalar-valued VDBs other than signed distance fields,"
             " use voxel values as local multipliers for point density."));
 
+    parms.add(hutil::ParmFactory(PRM_ORD, "poscompression", "Position Compression")
```

Comment:
Please use string-valued parameters for menus, so that the menu items can be reordered if necessary.

Reply:
Shouldn't it match SOP_OpenVDB_Points_Convert::poscompression? We wouldn't change one menu without also changing the other.

Reply:
Points Convert should also have used string-valued parameters.

Reply:
I agree. That being said, I think I'd rather it match, then we can go through the effort of changing both if we ever need to.

Reply:
OK, but then would it not be a good idea to add a function (or functions) to `PointUtils.h` to provide the mapping from integers to menu item strings, so that the two SOPs can't get out of sync?

Reply:
We haven't changed the options in this drop-down in at least five years and I don't have any plans to change it in the future, so that's not a big concern to me.